### ISSUE TITLE
optimizer

### ISSUE THREAD

**Issue by self-reward** [tsmotlp/DRGAN#4]
Hello, thanks for your work, but why you set double self.networks['netD_N'].parameters() in 'optim_D''（drgan_model.py）：
   ```
 'optim_D': torch.optim.Adam(itertools.chain(self.networks['netD_C'].parameters(),
                                                                               self.networks['netD_N'].parameters(),
                                                                               self.networks['netD_N'].parameters(),
                                                                               self.networks['netD_PN'].parameters()),
                                                               lr=self.args.lr, betas=(0.5, 0.999), weight_decay=1e-4)}
```
